src/client/proximaStreamClient.ts

```diff
@@ -133,11 +135,15 @@ export class ProximaStreamClient {
   private getStreamConsumerClient(endpoint: string) {
     return (
       this.clients[endpoint] ??
-      (this.clients[endpoint] = new StreamDBConsumerClient(endpoint))
+      (this.clients[endpoint] = new StreamDBConsumerClient(
```

Comment:
Is this the only place where the client is created internally? I thought that the registry client can also create a consumer client sometimes :thinking:

Reply:
only one as far as I can see